internal/controller/pkg/revision/reconciler_test.go

```diff
@@ -131,10 +131,13 @@ func (m *MockDependencyManager) RemoveSelf(ctx context.Context, pr v1.PackageRev
 	return m.MockRemoveSelf()
 }
 
+// Make sure not to use tabs instead of spaces in the providerBytes
```

Comment:
I added this comment because I learned the hard way that tabs don't work that well with the parser :sweat_smile:

Reply:
Oh okay, didn't know that. I will remove this then.